x-pack/plugins/transform/public/app/sections/transform_management/components/transform_list/expanded_row_messages_pane.tsx

```diff
@@ -84,13 +104,15 @@ export const ExpandedRowMessagesPane: React.FC<Props> = ({ transformId }) => {
       width: theme.euiSizeXL,
     },
     {
+      field: 'timestamp',
```

Comment:
Not essential for this PR, but how much work would it be to add a 'Refresh' button to the Messages tab, as we have for ML jobs?

![image](https://user-images.githubusercontent.com/7405507/175898468-24d086b7-a9c0-4fbb-ab70-e69b2d39d0bc.png)

Reply:
Updated here https://github.com/elastic/kibana/pull/135047/commits/4583bec88e4238075fcc68765c14aeb0af2f44bf